test/shared-class.test.js

```diff
@@ -143,12 +144,13 @@ describe('SharedClass', function() {
       SomeClass.prototype.myMethod = function() {};
       var METHOD_NAME = 'myMethod';
       sc.defineMethod(METHOD_NAME, {
+        isStatic: true,
```

Comment:
I could also use `isStatic: false` too, but in this case Line 153 needs to be changed to:

```
expect(sc.findMethodByName('prototype.myMethod').accessType).to.eql('READ');
```